NEWS

```diff
@@ -1,3 +1,67 @@
+Changes from 0.11.0
+
+This release brings some quite significant changes on the language and as a
+result very large portion of previously written Hy programs will require
+changes. At the same time, documentation and error messages were improved,
+hopefully making the language easier to use.
+
+   [ Language Changes ]
+   * New syntax for let, with and defclass
+   * defmacro will raise an error on &kwonly, &kwargs and &key arguments
+   * Keyword arguments to functions are required to be strings
+   * slice replaced with cut
+   * removed throw, catch, progn, defun, lisp-if, lisp-if-not and filterfalse
+   * global now takes multiple arguments
+   * Nonlocal keyword (Python 3 only)
+   * Set literals
+   * Keyword-only arguments
+   * Setv can assign multiple variables at once
+   * Empty form allowed for setv, del and cond
+   * One-argument division and rationals
+   * partition form for chunking collection to n-sized tuples
+   * defn-alias and demacro-alias moved into hy.contrib.alias
+   * Implicit nil in --init-- method
+   * for and cond can take multi-expression body
+   * Hex and octel support for integer literals
+   * Apply now mangles strings and keywords according to Hy mangling rules
+   * One arity comparison operators
+   * Variadic if
+   * defreader can use strings as macro names
+   * as-> macro added
+
+   [ Bug Fixes ]
+   * Better error when for doesn't have body
+   * Better error detection with list comprehensions in Python 2.7
+   * Setting value to callable will raise an error
+   * defclass can have properties / methods with built-in names
+   * Better error messages on invalid macro arguments
+   * Better error messages with hy2py and hyc
+   * Cmdline error to string conversion.
+   * In python 3.3+, generator functions always return a value
+   * &rest can be used after &optional
+
+   [ Misc. Improvements ]
+   * Allow specification of global table and module name for (eval ...)
+   * General documentation improvements
+   * Contrib.walk: Coerce non-list iterables into list form
+   * *args and **kwargs no longer have own args in Python 3.5
+   * Flow macros (case and switch)
+   * ap-pipe and ap-compose macros
+   * #@ reader macro for with-decorator
+   * Type check `eval` parameters
+   * `and` and `or` short-circuit
+   * `and` and `or` accept zero or more arguments
```

Comment:
This is e54d4be and the one in language changes is 0a942a0 (If I understood your intent correctly "One arity comparison operators"). I would keep them separate I guess.